### PR TITLE
add tooltip

### LINKED ISSUE #4: Add tooltip to range-slider
## Value Proposition

- As a user
- I would like to receive examples of how many grams correspond to which everyday quantities that I know
- to be able to better estimate how many grams of food I consumed

## Description

Image

## Acceptance criteria

- [ ]  The user can use the range slider to choose his consumption of it in grams
- [ ]  Directly below the range slider, the user gets information/a tooltip on everyday quantities to help estimate his
consumption
- [ ] For each food there should be about 3-4 breakpoint-bubbles, at which this info appears

## Tasks

- [ ] Create a new branch
&nbsp;
- [ ] Extent the foods-Array in the db.js with information on different everyday quantities for the foods
 &nbsp;
- [ ] Add functionality to the range-slider, so that when moving the slider, everyday quantities are shown using formData and the foods-Array (probably using an EventListener and threshold-logic)
&nbsp;
- [ ]  Write tests, testing different input to the range-slider and whether the tooltip correctly appears with the correct information

### PR DESCRIPTION
**Pull request by alinawinter**
fix #4

### CODE REVIEW

File: lib/db.js [hunk 1]
```diff
@@ -4,10 +4,27 @@ export const foodCategories = [
     name: "Vollkorngetreide",
     slug: "vollkorngetreide",
     recommendedConsumption: 232,
-    recommendedExample:
-      "2 Tassen Vollkornreis und 1,5 Scheiben Vollkornroggenbrot",
-    maxRange: 464,
-    maxRangeInputField: 600,
+    recommendedExample: "1 Tasse Vollkornreis und 1 Scheibe Vollkornroggenbrot",
+    maxRa600nge: 464,
```
Comment (augustinanders) [inline lib/db.js:8]: shouldn't this be `maxRange` instead of `maxRa600nge`?

**Review by augustinanders — APPROVED** [review]
Great job! Your app is already very extensive and impressive 🚀.

I've reviewed everything, but due to the extensive refactoring you've done, I may have missed some correlations. 

I left two comments for you - I hope they are helpful!

File: lib/db.js (hunk 1 above)
Comment (alinawinter) [reply lib/db.js:8]: yes!! Thanks alot!!

File: components/Form/FormQuantitySpecification/index.js [hunk 2]
```diff
@@ -51,10 +87,10 @@ export default function FormQuantitySpecification({
         onSubmit={handleQuantitySpecification}
       >
         <h3 id="form-title">Tägliches Essensquiz</h3>
```
[the remark this answers is not in the document]
Comment (alinawinter) [reply components/Form/FormQuantitySpecification/index.js:89]: oh thats a good point! will change it

**Comment by alinawinter** [comment]
Thank you very much @augustinanders !! :-)

**Pull request merged**